hello-world-php-task/readme.md

```diff
@@ -0,0 +1,8 @@
+# Hello World PHP
+
+Sample for Hello World task. Print "Hello World"
+
+## Getting started
+
+1. Fork this repository.
+2. Delpoy as a manual task in Choreo.
```

Comment:
Shall we add more details?

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/15/commits/add09046e6b8c2d071d9af8ab6d9b1722a30038e